README.md

```diff
@@ -199,3 +196,50 @@ It's likely that you'll be prompted with a screen with warnings about an unsafe
 There was an initial attempt to dockerize the IDP but it is currently deprecated, mostly non-functional, and not maintained. There is ongoing work to make the IDP more [12 Factor](https://12factor.net/) compliant which will eventually lead to better support for containerization.
 
 If you'd like to work with the previous implementation see the [Docker documentation](./docs/Docker.md) to install the IdP as a container.
+
+### Troubleshooting
+#### I am receiving errors when running `$ make setup`
+
+If this command returns errors, you may need to install the dependencies first, outside of the Makefile:
+```
+$ bundle install
+$ yarn install
+```
+
+#### I am receiving errors related to Capybara in feature tests
+You may need to install _chromedriver_ or your chromedriver may be the wrong version (`$ which chromedriver && chromedriver --version`).
+
+chromedriver can be installed using [Homebrew](https://formulae.brew.sh/cask/chromedriver) or [direct download](https://chromedriver.chromium.org/downloads). The version of chromedriver should correspond to the version of Chrome you have installed `(Chrome > About Google Chrome)`; if installing via Homebrew, make sure the versions match up.
+
+#### I am receiving errors when creating the development and test databases
+
+If you receive the following error (where _whoami_ == _your username_):
+
+`psql: error: connection to server on socket "/tmp/.s.PGSQL.5432" failed: FATAL:  database "<whoami>" does not exist`
+
+Running the following command first, may solve the issue:
+```
+$ createdb `whoami`
+```
+
+#### I am receiving errors when running `$ make test`
+
+##### Errors related to running specs in _parallel_
+`$ make test` runs specs in _parallel_ which could potentially return errors. Running specs _synchronously_ may fix the problem; to run specs _synchronously_:
+```
+$ make sync_test
+```
```

Comment:
ditto about sync vs serial

Also.... running the whole suite takes a very long time... maybe we could recommend running like an individual file or spec? (that's normally what I do, not the whole suite)